metaflow/plugins/project_decorator.py

```diff
@@ -26,6 +26,24 @@ class ProjectDecorator(FlowDecorator):
         projects that use the same production scheduler. The name may
         contain only lowercase alphanumeric characters and underscores.
 
+    branch : Optional[str]
+        The branch to use. If not specified, the branch is set to
+        `user.<username>` unless `production` is set to `True`. This can
```

Comment:
technically the branch is still <username>, the project branch name is user.<username>

Reply:
True, although not sure if we have explicitly documented `project_branch` anywhere in the user docs. For most intents and purposes, the value that's visible/exposed to the users is the `project_branch`.

Reply:
Right. From a user's pov, the stuff in between your project name and the flow name is user.<username> I'm happy to change the wording but thought that it would be nice to document somewhere.